VK/VK/Presentation/Groups/ViewControllers/GroupTableViewController.swift

```diff
@@ -81,12 +76,29 @@ final class GroupTableViewController: UITableViewController {
     }
 
     private func createCell(indexPath: IndexPath) -> UITableViewCell {
-        let model = groupsDataCourse[indexPath.row]
+        let model = groups[indexPath.row]
```

Comment:
```suggestion
        let group = groups[indexPath.row]
```
зачем объект назвала model если это 1 группа